src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -55,15 +67,19 @@
            (create-field name (display-value value)))
          log-map)])))
 
-
 (react/defc CallDetail
   {:get-initial-state
    (fn []
      {:expanded false})
    :render
    (fn [{:keys [props state]}]
      [:div {:style {:marginTop "1em"}}
-      [:div {:style {:display "inline-block" :marginRight "1em"}} (:label props)]
+      [:div {:style {:display "inline-block" :marginRight "1em"}} 
+       (let [workflowName (workflow-name (:label props)) callName (call-name (:label props))]
```

Comment:
Nits: `workflow-name` is more idiomatic for Clojure. We usually put each variable on its own line.